packages/components/src/Chip/Chip.test.tsx

```diff
@@ -49,13 +49,7 @@ describe("Chip", () => {
 
   it("should have a role of button when role not provided", () => {
     const { getByRole } = render(<Chip label="Test Chip" />);
-    expect(getByRole("button")).toBeInstanceOf(HTMLButtonElement);
-  });
-
-  it("should have a tabIndex of zero if one is not provided", () => {
```

Comment:
Chip no longer defaults to zero if you don't pass in a value, so this test is no longer valid.